cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -709,9 +709,7 @@ rebuildInstallPlan
           where
             corePackageDbs :: [PackageDB]
             corePackageDbs =
-              applyPackageDbFlags
-                [GlobalPackageDB]
-                (projectConfigPackageDBs projectConfigShared)
+              Cabal.interpretPackageDbFlags False (projectConfigPackageDBs projectConfigShared)
```

Comment:
Why is `GlobalPackageDB` not needed here? Is it already present in `projectConfigPackageDBs`?

Reply:
It is added by `interpretPackageDbFlags`.